sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -277,12 +306,17 @@ class wi_element {
   }
 
 #if __SYCL_DEVICE_ONLY__
+  // TODO: __spirv_VectorInsertDynamic should take storage element type as
+  // argument
 #define OP(op)                                                                 \
```

Comment:
Perhaps it is a good opportunity to remove all these macros?

Reply:
@yubingex007-a11y, I remember you change this code to use macros and make it more compact.
The code was before expanded for each of the ops. Bing changed it to remove the redundancy.
@keryell what do you suggest we should use instead?

Reply:
Sometimes macros are the best or only reasonable solution.
In that case use protected names like `__DPC_SYCL_OP` or whatever to avoid the case where a user decides to use in her program:
```c++
#define OP something
```
:-)